app/seasonal-planting-guide/page.tsx

```diff
@@ -59,67 +64,70 @@ export default function SeasonalPlantingGuide() {
 
   return (
     <PageContainer>
-      {!selectedUsState ? (
-        <>
-          <p>Please select a US state to view planting information.</p>
-          <StateOptionsContainer>
-            <FilterDropdownSingle
-              name="usState"
-              id="usState"
-              value={selectedUsState}
-              setStateAction={setSelectedUsState}
-              placeholder="US State"
-              options={usStateOptions}
-            />
-          </StateOptionsContainer>
-        </>
-      ) : (
-        <>
-          <HeaderContainer>
-            <SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
-            <FilterContainer>
-              <FilterDropdownSingle
-                name="usState"
-                id="usState"
-                value={selectedUsState}
-                setStateAction={setSelectedUsState}
-                placeholder="US State"
-                options={usStateOptions}
-              />
-
-              <FilterDropdownMultiple
-                value={selectedGrowingSeason}
-                setStateAction={setSelectedGrowingSeason}
-                options={growingSeasonOptions}
-                placeholder="Growing Season"
-              />
+      <HeaderContainer>
+        <PageTitle>
+          <H1 $color={COLORS.shrub} $align="left">
+            Planting Timeline
+          </H1>
+          <Icon type="info" />
```

Comment:
note: the updated designs will not an include the info icon on /seasonal-planting-guide (Sashank is creating the color-legend component which will be displayed directly on the page). However, we can keep the info svg in lib/icons.tsx as it will still be used in /view-plants